src/modules/commander/state_machine_helper.cpp

```diff
@@ -493,138 +493,8 @@ transition_result_t hil_state_transition(hil_state_t new_state, orb_advert_t sta
 			    || current_status->arming_state == vehicle_status_s::ARMING_STATE_STANDBY
 			    || current_status->arming_state == vehicle_status_s::ARMING_STATE_STANDBY_ERROR) {
 
-#ifdef __PX4_NUTTX
```

Comment:
Interesting stuff, so we don't want to use this blocking/unblocking in the future?